### ISSUE TITLE
`4_LED_Channels` processing fails

### ISSUE THREAD

**Issue by AbdulrhmnGhanem** [kitspace/kitspace-v2#376]
https://kitspace.org/boards/github.com/badenlab/led-zappelin/pcb/potentiometer_mounts/4_led_channels/

**Comment by AbdulrhmnGhanem** [comment]
We have three problems here:
1. sub-project names have slashes, already solved in #402.
2. The same race condition that happens in #392.
3. IBOM processing fails with the following error
```console
kitspace-processor-1  | failed tester/led-zappelin/dc5a1fca1df29ebe028df02473966ed723d06e5a/PCB-Potentiometer-Mounts-24-LED-Channels/interactive_bom.json Command failed: /app/src/tasks/processIBOM/run_ibom '/data/checkout/tester/led-zappelin/PCB/Potentiometer_Mounts/16_LED_Channels/Potentiometer_mount_16LED.kicad_pcb' 'PCB-Potentiometer-Mounts-24-LED-Channels' '24 LED channels option board for the LED Zappelin' project' '/data/files/tester/led-zappelin/dc5a1fca1df29ebe028df02473966ed723d06e5a/PCB-Potentiometer-Mounts-24-LED-Channels/interactive_bom.json'
kitspace-processor-1  | /bin/sh: 1: Syntax error: Unterminated quoted string
```

**Comment by AbdulrhmnGhanem** [comment]
> 3. IBOM processing fails with the following error
```console
kitspace-processor-1  | failed tester/led-zappelin/dc5a1fca1df29ebe028df02473966ed723d06e5a/PCB-Potentiometer-Mounts-24-LED-Channels/interactive_bom.json Command failed: /app/src/tasks/processIBOM/run_ibom '/data/checkout/tester/led-zappelin/PCB/Potentiometer_Mounts/16_LED_Channels/Potentiometer_mount_16LED.kicad_pcb' 'PCB-Potentiometer-Mounts-24-LED-Channels' '24 LED channels option board for the LED Zappelin' project' '/data/files/tester/led-zappelin/dc5a1fca1df29ebe028df02473966ed723d06e5a/PCB-Potentiometer-Mounts-24-LED-Channels/interactive_bom.json'
kitspace-processor-1  | /bin/sh: 1: Syntax error: Unterminated quoted string
`````
Fixed in #407.
